mlflow/tensorflow.py

```diff
@@ -584,8 +584,8 @@ def _log_event(event):
     """
     if not mlflow.active_run():
         try_mlflow_log(mlflow.start_run)
```

Comment:
Should we pull this logic for checking if a run is active & creating one if not into a helper like `_maybe_create_active_run`? Would make it easier to standardize